source/documentation/doc-Release_Notes/topics/ref-Removed_Features_RHV.adoc

```diff
@@ -61,8 +61,10 @@ Removing this neither affects the ability to manage {virt-product-fullname} virt
 
 | Cockpit installation for Self-hosted engine| Using Cockpit to install the self-hosted engine is no longer supported. Use the command line installation.
 
-// Uncomment this line for RHV release that corresponds to oVirt 4.5.
-// Also, remove Deprecation notice for this feature for same release.
-// | oVirt Scheduler Proxy | The `ovirt-scheduler-proxy` package is removed in {virt-product-fullname} 4.4.z SP1.
-
+ifdef::rhv-doc[]
+| oVirt Scheduler Proxy | The `ovirt-scheduler-proxy` package is removed in {virt-product-fullname} 4.4 SP1.
+endif::rhv-doc[]
+ifdef::ovirt-doc[]
```

Comment:
since oVirt has their own independent release notes, this is not needed - our Release Notes docs only apply to RHV/downstream

Reply:
ok